src/Stratis.Bitcoin/Base/Deployments/ThresholdConditionCache.cs

```diff
@@ -85,8 +86,10 @@ public List<ThresholdStateModel> GetThresholdStateMetrics(ChainedHeader indexPre
                 int currentHeight = indexPrev.Height + 1;
                 int period = this.consensus.MinerConfirmationWindow;
 
-                // First ancestor outside last confirmation window.
-                ChainedHeader periodStartsHeader = indexPrev.GetAncestor(indexPrev.Height - (currentHeight % period));
+                // First ancestor outside last confirmation window. If we haven't reached block height 2016 yet this will be the genesis block.
```

Comment:
`GetAncestor()` was returning null because we were asking for block -1 when the chain was at a low height.